pkg/component/component_full_description.go

```diff
@@ -185,10 +194,21 @@ func NewComponentFullDescriptionFromClientAndLocalConfigProvider(client kclient.
 	}
 	cfd.Spec.URL = urls
 
-	err = cfd.loadStoragesFromClientAndLocalConfig(client, configProvider, componentName, applicationName, &componentDesc)
+	// Obtain Storage information
+	var storages storage.StorageList
+	storageClient := storage.NewClient(storage.ClientOptions{
+		Client:              client,
+		LocalConfigProvider: envInfo,
```

Comment:
Something's wrong here. `LocalConfigProvider` was created for `.odo/config.yaml`. For `envInfo`, we should be using `EnvSpecificInfo` or something like that. Has that changed recently?

Reply:
LocalConfigProvider is the interface, EnvSpecificInfo implements this interface. We did not change this name thinking that it made sense if we wanted to write a different implementation.